README.md

```diff
@@ -1,2 +1,15 @@
-# sdk-code-gen
-LibLab.com SDK Code Generator 
+# SDK Code Generator (Coming Soon)
+
+Your API deserves a better SDK. Instantly generate SDKs in multiple languages for your API service.
+
+```console
+> liblab generate-sdk --api="openapi.yaml"
+Generating SDK for Java...done ✓
+Generating SDK for Python...done ✓
+Generating SDK for JavaScript...done ✓
+Generating docs for developer portal...done ✓
+```
+
+[Join Our Community](https://discord.gg/F8aECHbRkV)
+
+Follow and Star this Repository to get the latest changes and updates as we airdrop our Beta MVP in the near future.
```

Comment:
I re-read this. Thinking we don't need to explicitly mention "MVP". I suggest updating it to "Beta launch".

Reply:
There is already a link back to liblab.com in the About section of the Repo and updated all of the above.